examples/model_building/plot_bcic_iv_2a_moabb_cropped.py

```diff
@@ -158,6 +158,9 @@
     final_conv_length=30,
 )
 
+# Display torchinfo table describing the model
+print(model)
```

Comment:
is it the __str__ or __repr__ displayed here?

Reply:
it's gonna be `str` as print first tries the `__str__` method and if it does not exist that fallbacks to `__repr__`

Reply:
we may call `str` explicitly in case you feel it would be easier to read